roles/kubernetes/control_plane/init/primary/templates/kubeadm-config.j2

```diff
@@ -51,11 +62,15 @@ cgroupDriver: systemd
 #
 # kube-proxy Configuration : https://kubernetes.io/docs/reference/config-api/kube-proxy-config.v1alpha1/
 #
+{% if kubernetes_version is version('1.31.0', 'lt', version_type='semver') %}
 apiVersion: kubeproxy.config.k8s.io/v1alpha1
+{% else %}
+apiVersion: kubeproxy.config.k8s.io/v1beta1
+{% endif %}
```

Comment:
There seem to be lots of additions here that are not in the current docs. Please provide info as to where you're finding these apis vs the official doc here - https://kubernetes.io/docs/reference/config-api/kube-proxy-config.v1alpha1/ I believe I am using the latest docs from kubernetes.